src/app/workflow/snapshot-exporter-modal/snaphot-exporter-modal.component.html

```diff
@@ -139,47 +144,65 @@ <h4 mat-dialog-title>
     </ol>
   </ng-template>
 
-  <ng-template #zenodoCheck>
-    <div
-      *ngIf="(hasZenodoToken$ | async) === false && !version.doiURL"
-      class="alert-danger p-2"
-      fxLayout="row"
-      fxLayoutGap="0.625rem;"
-      fxLayoutAlign="end top"
-    >
-      <div>
-        <mat-icon class="alert-danger-icon">warning</mat-icon>
+  <ng-template #zenodoNotLinked>
+    <div>
+      <div data-cy="zenodo-not-linked">
+        To request a DOI, you must first link your Zenodo account in your Dockstore Linked Accounts section. After linking, come back to
+        this screen and try again.
       </div>
       <div>
-        <div data-cy="zenodo-not-linked">
-          To request a DOI, you must first link your Zenodo account in your Dockstore Linked Accounts section. After linking, come back to
-          this screen and try again.
-        </div>
-        <div>
-          <button data-cy="link-zenodo" class="ml-1 accent-1-dark mat-elevation-z" mat-raised-button (click)="linkAccount()">
-            <mat-icon>link</mat-icon>
-            Link account now
-          </button>
-        </div>
+        <button data-cy="link-zenodo" class="ml-1 accent-1-dark mat-elevation-z" mat-raised-button (click)="linkAccount()">
+          <mat-icon>link</mat-icon>
+          Link account now
+        </button>
       </div>
     </div>
   </ng-template>
 
-  <ng-template #orcidCheck>
-    <div *ngIf="(hasOrcidToken$ | async) === false" class="alert-danger p-2" fxLayout="row" fxLayoutGap="0.625rem;" fxLayoutAlign="end top">
-      <div>
-        <mat-icon class="account-warning alert-danger-icon">warning</mat-icon>
+  <ng-template #orcidNotLinked>
+    <div>
+      <div data-cy="orcid-not-linked">
+        To export a workflow to ORCID, you must first link your ORCID account in your Dockstore Linked Accounts section. After linking, come
+        back to this screen and try again.
+        <button data-cy="link-orcid" class="ml-1 accent-1-dark mat-elevation-z" mat-raised-button (click)="linkAccount()">
+          <mat-icon>link</mat-icon>
+          Link account now
+        </button>
       </div>
+    </div>
+  </ng-template>
+
+  <ng-template #noAuthors>
+    <div>
+      <div data-cy="no-authors">
+        Before snapshotting, you should ensure your version has its author(s) set. The authors cannot be added after you have snapshotted
+        the version. You can either add authors via
+        <a
+          href="{{
+            Dockstore.DOCUMENTATION_URL
+          }}/assets/templates/tools/tools.html?highlight=authors#filled-out-example-of-multiple-tools-in-the-same-repository"
+          target="_blank"
+          rel="noopener noreferrer"
+          >the .dockstore.yml</a
+        >, which is preferred if you have ORCID ids for the authors, or by adding them
```

Comment:
ORCID wants us to call it `ORCID iDs` instead of `ids` https://info.orcid.org/brand-guidelines/